vrk-kpa/suomidigi
# SUOPA-774 Blog post redirects

To test: 
- `make fresh`
-  Import the redirects from Suomidigi WP:
   - `drush migrate-import suomidigi_wordpress_redirects --feedback="50 items"`
- Go to https://suomidigi.docker.sh/admin/config/search/redirect and check that there are new redirects -->  Compare the dates. :)
   - Check that the items on "Osoitteeseen" column has complete path alias. Something like "/blogit/digitaalinen-oppiminen-nyt"
- Also test that the redirects work, by copying the item from "Lähettäjä" column to your browser (of course with a https://suomidigi.docker.sh -prefix)
